src/Symfony/Component/Workflow/composer.json

```diff
@@ -26,7 +26,8 @@
         "psr/log": "~1.0",
         "symfony/event-dispatcher": "~2.1|~3.0",
         "symfony/property-access": "~2.3|~3.0",
-        "twig/twig": "~1.14"
+        "twig/twig": "~1.14",
+        "phpunit/phpunit": "^5.3"
```

Comment:
I reverted that. Symfony does not work like that.
And so I also reverted `getMock` -> `createMock`